hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java

```diff
@@ -92,7 +92,7 @@ public String generate(Config config) {
            .append("xmlns=\"http://www.hazelcast.com/schema/config\"\n")
            .append("xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n")
            .append("xsi:schemaLocation=\"http://www.hazelcast.com/schema/config ")
-           .append("http://www.hazelcast.com/schema/config/hazelcast-config-3.9.xsd\">");
+           .append("http://www.hazelcast.com/schema/config/hazelcast-config-3.10.xsd\">");
         gen.open("group")
```

Comment:
You need to add the flake ID config generation here. You can then also add a test to `ConfigXmlGeneratorTest`.